docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>Do we need any comment here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>This was included by mistake. Removed.</p>